test/test_transforms_v2.py

```diff
@@ -82,6 +82,30 @@ def auto_augment_adapter(transform, input, device):
     return adapted_input
 
 
+def auto_augment_detection_adapter(transform, input, device):
+    adapted_input = {}
+    image_or_video_found = False
+    bounding_box_found = False
+    for key, value in input.items():
+        if isinstance(value, datapoints.Mask):
+            # AA detection transforms don't support masks
+            continue
+        elif isinstance(value, datapoints.BoundingBox):
+            # TODO: this will only test the first bbox
+            if bounding_box_found:
+                # AA detection transforms only support a single bounding box tensor
+                continue
+            bounding_box_found = True
```

Comment:
We may want to test multiple BoundingBox Formats

Reply:
Why can't we have mutliple bboxes ?
As for bbox format yes we can test on multiple formats but bbox format it is a detail.